### PR TITLE
New user guides 0322

### PR DESCRIPTION
**Pull request by tutacrypto**

### CODE REVIEW

**Comment by tutacrypto** [comment]
need to do a rebase

File: app-guides/mint-burn.md [hunk 1]
```diff
@@ -2,35 +2,50 @@
...
+agEUR is a **EUR-pegged stablecoin**, meaning that it's built to mirror the value of the EUR. 
 
-To interact with Angle protocol, you will need to approve tokens for each collateral/stablecoin pool. You can learn more about approving tokens [here](app-faq/#why-do-i-need-to-approve-the-same-token-multiple-times).
+If you are new to crypto, follow our [guide for new crypto users](/app-guides/newbie.md). Then, keep reading here to see how you can get agEUR on the blockchain. 
+
+There are multiple ways to get agEUR: 
+
+- mint new agEUR directly from the [Angle App](https://app.angle.money/) against your tokens
+- swap ETH for agEUR on [Uniswap](https://app.uniswap.org)
+- buy agEUR with credit card from one of our on-ramp partner like [Wert.io](https://widget.wert.io/01FPZ4G1VMZ67MGC83NADPB0F8/redirect/?commodity=agEUR&commodities=agEUR,agEUR%3Aethereum)
```
Comment (sogipec) [inline app-guides/mint-burn.md:15]: one of our on-ramp partners

File: app-guides/mint-burn.md [hunk 2]
```diff
@@ -2,35 +2,50 @@
...
+You now have agEUR that mirror the value of the EUR!
 
 ![Minting agEUR](../.gitbook/assets/mint-usdc-agEUR.png)
 
-## Burning
+### Burning agEUR
+
+If you want to get rid of your agEUR, you have a few options: 
+- swap them against another token on [Uniswap](https://app.uniswap.org)
+- burn agEUR against one of the available token from the [Angle App](https://app.angle.money/#/user/burn).
```
Comment (sogipec) [inline app-guides/mint-burn.md:43]: one of the available tokens

File: app-guides/newbie.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Now that you have a wallet and your seed phrase is safe, you will need ether (ETH) to pay for transaction fees on the blockchain (called gas). You can read more about that [here](https://metamask.zendesk.com/hc/en-us/articles/360015489611-Learn-the-basics-of-blockchains-and-Ethereum-miners-and-validators-gas-cryptocurrencies-and-NFTs-block-explorer-networks-etc-). 
+
+You can buy ETH directly from Metamask by following [this](https://metamask.zendesk.com/hc/en-us/articles/360058239311-Directly-buying-tokens-with-on-ramps-in-MetaMask
+) guide. You will need at least 0.1 ETH to pay for a few transactions. 
+
+If you have an account at a centralized exchange, you can also buy ETH from there and transfer it to one of your Metamask account. 
+
+### Follow our Angle guides
+
+Once you have ETH in your wallet, you can follow our Angle guides to see how to use the protocol.
```
Comment (sogipec) [inline app-guides/newbie.md:43]: Let's add a link to a guide. 

Or now that you're setup, you may want to start having fun and getting yield on Euros. Take a look for example at this guide we have made to get agEUR

File: app-guides/slp-pools.md [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
 
 1. Choose the pool you would like to deposit your tokens in and click on `Deposit`.
-2. On the deposit page, enter the amount of tokens you want to put in the pool and confirm the transaction. You might need to approve the pool if it hasn't been done before.
-3. You will receive sanTokens in exchange, which represent your liquidity in the pool and automatically accrue interest.
-4. To withdraw your funds, just click on the `Withdraw` button on the Pools page, and input the number of sanTokens you want to send back.
+2. On the deposit page, enter the amount of tokens you want to put in the pool. You might need to approve the token through a transaction or a signature. 
+3. You now have two options: `Deposit`, and `Deposit and Stake`. 
+    - **`Deposit`** will send your collateral to the pool, and you will receive the associated sanTokens, which automatically accrue interest. 
+    - **`Deposit and Stake`** will deposit your collateral into the protocol, generate the associated sanTokens, and stake them automatically in the related gauge on the [DAO App](https://dao.angle.money/#/stake). You will start accumulating ANGLE rewards on top of the interest of the sanTokens.
```
Comment (sogipec) [inline app-guides/slp-pools.md:15]: You are not staking on the DAO App: you are staking in the liquidity gauge which you can track on the DAO App

File: veangle-guides/incentives.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+description: Earning ANGLE rewards
+---
+
+# Earning ANGLE rewards
+
+To earn ANGLE distributed as part of the weekly emissions, users can deposit sanTokens, open long positions on the protocol with Perps, or deposit LP (liquidity providers) tokens. 
+
+You can find all the available pools in the [Incentives page](https://dao.angle.money/#/stake) of the DAO App. 
+
+After staking your tokens, you will start accruing ANGLE. You can claim your tokens from each card, or from the `Claim Rewards` button on your left to claim all rewards from Perps and boosted gauges in one transaction.
```
Comment (sogipec) [inline veangle-guides/incentives.md:11]: Let's add a screenshot and say that you can decide not to claim from a particular gauge/perp if you feel that you have not accumulated enough rewards. The more gauges/things you'll claim the more expensive the transaction is going to be

File: veangle-guides/locking-angle.md [hunk 6]
```diff
@@ -34,10 +34,19 @@ To compute the balance of veANGLE needed to get the max boost on rewards dependi
...
-![Increasing or adding tokens to lock](../.gitbook/assets/increase-add-lock.png)
+![Increasing or adding tokens to lock](../.gitbook/assets/claim-interest.png)
 
-### Claiming interest
+## Claiming interest
 
 Every week, a share of the interest generated by the protocol is redistributed with veANGLE holders. These interest are currently being distributed in the form of sanUSDC\_EUR tokens, earning yield from the protocol. They can always be redeemed for USDC from the protocol.
 
-These interest can be claimed with a transaction initiated by clicking on the **Claim Interest (or Claim Fees)** button on the bottom right corner.
+These interest can be claimed with a transaction initiated by clicking on the `Claim Interest` button on the bottom right corner of the above screnshot. Since the router release, users also have the options to `Claim and Stake`, and `Claim and Mint agEUR`
```
Comment (sogipec) [inline veangle-guides/locking-angle.md:43]: We can add a screenshot to detail the different options here

**Comment by tutacrypto** [comment]
@sogipec can be merged

**Pull request merged**